compiler/src/dotty/tools/dotc/typer/QuotesAndSplices.scala

```diff
@@ -185,6 +171,21 @@ trait QuotesAndSplices {
       tree1
   }
 
+  def typedQuotedTypeVar(tree: untpd.Ident, pt: Type)(using Context): Tree =
```

Comment:
A documentation comment for this method would be nice.

Reply:
I will add it